src/Panel/Model.php

```diff
@@ -387,6 +387,32 @@ public function toLink(string $tooltip = 'title'): array
         ];
     }
 
+    /**
+     * Returns link url and tooltip
+     * for optional sibling model and
+     * preserves tab selection
+     *
+     * @internal
+     *
+     * @param \Kirby\Cms\ModelWithContent|null $model
+     * @param string $tooltip
+     * @return array
+     */
+    protected function toPrevNextLink($model = null, string $tooltip = 'title'): ?array
+    {
+        if ($model === null) {
+            return null;
+        }
+
+        $data = $model->panel()->toLink($tooltip);
+
+        if ($tab = get('tab')) {
+            $data['link'] .= '?tab=' . $tab;
```

Comment:
This is more Kirby style 🙂 

```suggestion
            $uri = new Uri($data['link'], [
                'query' => ['tab' => $tab]
            ]);

            $data['link'] = $uri->toString();
```